eggbox_potential_sampler/random_sampling_mco/tests/test_example_mco_factory.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+
+# from force_bdss.core_plugins import BasicMCOCommunicator
+from enthought_example.example_mco.example_mco_communicator import \
```

Comment:
We should probably go with the bracketed list of imports since we have one a few lines down. I think they recommend doing that even for lists of 1!